darp 11 windows drivers

garrettjwilke opened the pull request.

garrettjwilke committed:
Add darp11 drivers

crawfxrd commented:
Where did these come from?

garrettjwilke commented:
> Where did these come from?
> 
> Folder should probably be named `arl-hu`/`arl-uh`, as Intel makes a separate kit for S/HX drivers.

i wasn't sure about the folder name. i will change to `arl-hu`.
these are from intel site except for the serialIO driver which i got from a giant 5GB ISO from sager:
https://www.sagernotebook.com/drivers.php?cat=855

crawfxrd commented on README.md:
```diff
@@ -15,6 +15,7 @@ there, please contact support for Windows driver information.
 - [Darter Pro (darp8)](#darter-pro-darp8)
 - [Darter Pro (darp9)](./drivers/intel/rpl-uph)
 - [Darter Pro (darp10)](./drivers/intel/mtl-uh)
+- [Darter Pro (darp11)](./drivers/intel/arl)
```
update link

crawfxrd commented:
Are the public drivers sufficient or do we still want to use what's in the BKC kit from Intel developer resources?

garrettjwilke commented:
everything worked on both darp11 and darp11-b.
is there anything specific i should check?

crawfxrd commented:
Most of the archives from the BKC have no installers anymore, and they also now contain PDBs. What that fuck is Intel doing with these?

---

- Provide links to the Intel site for each driver
  - At least in the commit message, but might be useful to add in a file in `drivers/intel`
- There is a `WiFi` and a `Wifi` zip
  - `Wifi` has the extract `darp11-wifi`; did you make this? Should it be removed?

crawfxrd committed:
arl-hu: Use serial driver from older BKC

Grab the driver from this kit before Intel deletes it later this year.

- Kit: 830041
- Version: WW33
- Updated: 2024-08-10

This version still has `SetupSerialIO.exe`. The "v2" repack present in
all later BKC kit versions does not have the installer.

Signed-off-by: Tim Crawford <tcrawford@system76.com>

garrettjwilke commented:
working:

- [x] wifi driver
- [x] serial IO
- [x] thunderbolt

leviport reviewed (approved):
Approving based off Garrett's testing, since he can't approve his own PR.

crawfxrd reviewed (approved):
I don't care anymore.
